app/javascript/src/get-snippet.js

```diff
@@ -35,6 +35,6 @@ function loadSnippet(event, element) {
     })
     .catch(error => {
       console.error(error)
-      ideElement.innerHTML = `Failed to load Snippet <br>${error}`
+      ideElement.innerText = `Failed to load Snippet <br />${error}`
```

Comment:
Wouldn't this cause the `<br />` to be set as part of the text?

Reply:
Ah yup, of course